test/v2/services/MessagesXmlParsingServiceSpec.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.services
+
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.must.Matchers
+import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
+import v2.base.TestActorSystem
+import v2.base.StreamTestHelpers
+import v2.models.errors.FailedToValidateError
+import v2.models.request.MessageType
+
+import scala.xml.NodeSeq
+
+class MessagesXmlParsingServiceSpec
+    extends AnyFreeSpec
+    with TestActorSystem
+    with Matchers
+    with StreamTestHelpers
+    with ScalaFutures
+    with ScalaCheckPropertyChecks {
+
+  val validXml: NodeSeq =
+    <CC013C>
```

Comment:
I'm trying to standardise this across everything, as the actual root node that comes from a trader will look like:

```xml
<ncts:CC013C PhaseID="NCTS5.0" xmlns:ncts="http://ncts.dgtaxud.ec">...</ncts:CC013C>
```

so please use that in your tests, rather than just the root name (it shouldn't affect the tests, but it's better to have this so that if something changes, our tests will fail and we'll know about it).

Reply:
done